app/models/recurring_task_config.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class RecurringTaskConfig
+
+  # Returns an array of recurring task classes and their calling methods.
+  # Engines can append to this list.
+  def self.recurring_tasks
+    @@recurring_tasks ||= [
+      [AutoExpireReservation, :perform],
```

Comment:
Some of these were running every 5 minutes, but in testing they are pretty fast (<1 second each) and I couldn't see any problem in running them more frequently, other than reducing the general load.  I could see adding an element here to configure cadence like:
`[AutoExpireReservation, :perform, 5]`
... and then we could maintain the same behavior as before.  Thoughts?